third_party/blink/renderer/platform/context_lifecycle_observer.h

```diff
@@ -41,6 +45,13 @@ class PLATFORM_EXPORT ContextLifecycleObserver : public GarbageCollectedMixin {
 #endif
 };
 
+// RUN-1716
+typedef HeapObserverSet<
+    ContextLifecycleObserver,
+    HeapHashSet<WeakMember<ContextLifecycleObserver>,
+                WTF::MemberHashRecordReplayId<ContextLifecycleObserver>>>
+    ContextLifecycleHeapObserverSet;
+
```

Comment:
Tiny nit: can you make this a `using` type alias? I find it a little bit easier to figure out which name is being introduced.

Reply:
Done! Would you say that `using` is always better than `typedef`?
(it looks like `typedef` is a C'ism, is it?)

Reply:
I prefer `using` because it's a little bit more general (mostly it can be templated) and I think it's a little bit more natural to read. Tiniest of nit (and yeah, `typedef` is a C'ism).